dev-infrastructure/docs/development-setup.md

```diff
@@ -259,6 +265,13 @@ Option 2: You can follow the below manual steps from the root of the CS repo on
 # Setup the development.yml
 cp ./configs/development.yml .
 
+# Setup the azure-runtime-config.json
+cp ./configs/azure/example-config.json ./azure-runtime-config.json
+
+#  Get azure-first-party-application-client-id
+CLIENT_ID=$(az ad app list --display-name aro-hcp-dev-first-party --query '[*]'.appId -o tsv)
```

Comment:
I am not sure that this display name is correct anymore. @janboll @JameelB could you check?

Aside from that, should we filter by display name or just specify the client id?

Reply:
that's not the correct one any longer, I think the display name is `aro-dev-first-party`. The new client id is 57e54810-3138-4f38-bd3b-29cb33f4c358

If we specify the client id here and it changes, we would need to remember to update this doc too. Might be less likely for the name to change?